library/src/main/scala/JavaCodeGen.scala

```diff
@@ -72,6 +74,7 @@ class JavaCodeGen(lazyInterface: String, optionalInterface: String) extends Code
       s"""${genPackage(e)}
          |${genDoc(doc)}
          |public enum $name {
+         |    ${extra mkString EOL}
```

Comment:
I think this has to follow the values, and you need a semi-colon as well.. but this is just from memory, I could be miss-remembering.